### ISSUE TITLE
Error on pg.init

### ISSUE THREAD

**Issue by creatorrr** [CanopyTax/asyncpgsa#17]
On trying to invoke `pg.init`, the module throws this cryptic error. I have been looking for this `init` required keyword in the documentation but could not find any thing. Am I missing something? Please help.

The code that triggered it:

```python
import asyncio
from asyncpgsa import pg

if __name__ == "__main__":

    loop = asyncio.get_event_loop()
    loop.run_until_complete( pg.init( dsn="postgres:///tranql_bot" ) )

```

And error:
> Traceback (most recent call last):
>   File "/usr/lib/python3.5/runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "/home/diwank/github.com/creatorrr/tranql-bot/app/__main__.py", line 12, in <module>
>     loop.run_until_complete( pg.init( dsn="postgres:///tranql_bot" ) )
>   File "/usr/lib/python3.5/asyncio/base_events.py", line 457, in run_until_complete
>     return future.result()
>   File "/usr/lib/python3.5/asyncio/futures.py", line 292, in result
>     raise self._exception
>   File "/usr/lib/python3.5/asyncio/tasks.py", line 239, in _step
>     result = coro.send(None)
>   File "/home/diwank/.envs/tranql/lib/python3.5/site-packages/asyncpgsa/pgsingleton.py", line 33, in init
>     self.__pool = await create_pool(*args, **kwargs)
>   File "/home/diwank/.envs/tranql/lib/python3.5/site-packages/asyncpgsa/pool.py", line 75, in create_pool
>     **connect_kwargs)
> TypeError: __init__() missing 1 required keyword-only argument: 'init'
>

**Comment by nhumrich** [comment]
Looks like this is an issue from an update to asyncpg. I should be locking versions, but I am not. I will get this fixed for you, but for now you can lock asyncpg to 0.8.4

**Comment by nhumrich** [comment]
Ok, this is fixed in the latest release: 0.8.1 of asyncpgsa.

Sorry for the confusion.
